tea-config-generator/serverless-sandbox.yml

```diff
@@ -0,0 +1,68 @@
+service: tea-config-generator
```

Comment:
I don't think we want a completely different deployment for sandbox. What would be the point? I think we can assume that CMR is deployed in Sandbox before TEA is deployed and document it properly.

Reply:
I can get rid of it, I just found it helpful for testing some things before messing with the SIT AWS configuration.

Reply:
I see the value of being able to test TEA configuration outside of CMR in Sandbox though. So I am OK with keeping it, but please add some documentation in the README.